src/cider/nrepl/middleware/debug.clj

```diff
@@ -435,14 +445,13 @@
       (try
         (read-string code)
         (catch Exception e)))
-    (binding [*skip-breaks* (atom nil)]
```

Comment:
I agree with this change, but PRs are easier to read if unrelated changes are left in separate commits. ;-)